servicetalk-http-api/src/main/java/io/servicetalk/http/api/StrategyInfluencerAwareConversions.java

```diff
@@ -19,13 +19,27 @@
 
 import static java.util.Objects.requireNonNull;
 
-final class StrategyInfluencerAwareConversions {
+/**
+ * Utility for converting filter factories to conditional factories which are applied when provided
+ * {@link Predicate} returns {@code true}. The returned implementations comply to the
+ * {@link HttpExecutionStrategyInfluencer} hint interface and apply chosen
+ * {@link io.servicetalk.transport.api.ExecutionStrategy} when invoked.
+ */
+public final class StrategyInfluencerAwareConversions {
```

Comment:
Summary from offline discussion: we don't need to make this utility class public. Methods that are required for the server builder impl can be moved to `http-netty` as pkg-private.